### ISSUE TITLE
xlsx problem on Linux using checkTasks

### ISSUE THREAD

**Issue by tpfau** [SysBioChalmers/RAVEN#17]
Hi,
When running the checkTasks on Linux using e.g. the files provided by Blais et al ( http://www.nature.com/articles/ncomms14250 ), I run into troubles that the file cannot be read. Would it be possible to add an option to use tab separated value files instead of xls here? 
I assume this is not an issue if there is a working excel installation on a windows machine, but even on windows this makes the toolbox reliant on excel being present.

**Comment by edkerk** [comment]
I had a quick look at this, sounded easy to fix. parseTaskList should be adjusted to also allow to read tab-delimited files. However, Matlab doesn't seem to have an good working equivalent for xlsread(), importdata() should work, but it finds the combination of text and numbers very confusing.
Regardless, according to xlsread() documentation:

> If your system does not have Excel for Windows, xlsread operates in basic import mode, and reads only XLS, XLSX, XLSM, XLTX, and XLTM files.

So it shouldn't be a problem. What error message do you get when reading the file?

**Comment by tpfau** [comment]
Hi,
I just found out part of the error, I expected a # to make the parser ignore the first line, but it did not, so I had to remove the line (which was the supplementals description).
In addition, when working with Openoffice, I encountered an issue where matlab told me that it did not understand the file format. Modifying the file with excel worked in the end.
But I would still suggest to alter the code in a way that allows the use of CSVs in the tasks definition (avoids odd file format issues).

Best,
Thomas
